### PR TITLE
xfstests: Init xfstests in alp main file

### PR DESCRIPTION
**Pull request by SeroSun**
Init xfstests in alp main file, and makes it install success remove fio package first, because one of fio dependency build fails in alp.

- Related ticket: https://progress.opensuse.org/issues/132533
- Verification run: 

> Use create_hdd_xfstests to create qcow2 which xfstests installed, and runs other tests based on that as  HDD_1
> - create_hdd_xfstests http://10.67.133.133/tests/278
> - example for normal sub tests:
> - xfstests_xfs-generic-001-100 http://10.67.133.133/tests/280
> - xfstests_xfs-xfs-001-100 http://10.67.133.133/tests/281
> - xfstests_btrfs-btrfs-001-050 http://10.67.133.133/tests/282
> - xfstests_btrfs-generic-001-100 http://10.67.133.133/tests/283
> - xfstests_ext4-ext4 http://10.67.133.133/tests/284
> - xfstests_ext4-generic-001-100 http://10.67.133.133/tests/285

### CODE REVIEW

File: tests/xfstests/install.pm [hunk 1]
```diff
@@ -45,7 +45,7 @@ sub install_xfstests_from_repo {
     zypper_call('--gpg-auto-import-keys ref');
     record_info('repo info', script_output('zypper lr -U'));
     if (is_transactional) {
-        trup_call('pkg install xfstests fio');
+        trup_call('pkg install xfstests');
```
Comment (ricardobranco777) [inline tests/xfstests/install.pm:48]: Why not remove `fio` only for ALP testing for it with `is_alp` ?

Also, I would `record_soft_failure` not having `fio` in ALP.

File: tests/xfstests/install.pm [hunk 2]
```diff
@@ -45,7 +45,12 @@ sub install_xfstests_from_repo {
     zypper_call('--gpg-auto-import-keys ref');
     record_info('repo info', script_output('zypper lr -U'));
     if (is_transactional) {
-        trup_call('pkg install xfstests fio');
+        if (is_alp) {
```
Comment (jlausuch) [inline tests/xfstests/install.pm:48]: minor comment, feel free to ignore:
```
trup_call('pkg install xfstests');
unless (is_alp) {
    trup_call('--continue  pkg install fio');
}
```

File: lib/main_micro_alp.pm [hunk 3]
```diff
@@ -277,6 +277,23 @@ sub load_slem_on_pc_tests {
     loadtest("publiccloud/ssh_interactive_end", run_args => $args);
 }
 
+sub load_xfstests_tests {
+    loadtest 'microos/disk_boot';
```
Comment (jlausuch) [inline lib/main_micro_alp.pm:281]:
```suggestion
    load_boot_from_disk_tests;
```

File: tests/xfstests/install.pm (hunk 2 above)
Comment (SeroSun) [reply tests/xfstests/install.pm:48]: +1, this is better xD

File: lib/main_micro_alp.pm (hunk 3 above)
Comment (SeroSun) [reply lib/main_micro_alp.pm:281]: ah, there has a standard function. I'll modify it. thx!

**Review by ricardobranco777 — APPROVED** [review]
LGTM

**Review by jlausuch — APPROVED** [review]
LGTM!
Btw, is there an issue if `fio` is not available in SLE Micro?

**Comment by SeroSun** [comment]
> LGTM! Btw, is there an issue if `fio` is not available in SLE Micro?
It would also fail in SLE Micro if it has a dependency issue, but I'm not sure, I don't have a test machine in SLE Micro. Actually I didn't add it into SLE Micro test yet, l will do more tests before xfstests enable in SLE Micro.

File: tests/xfstests/install.pm [hunk 4]
```diff
@@ -45,7 +45,10 @@ sub install_xfstests_from_repo {
     zypper_call('--gpg-auto-import-keys ref');
     record_info('repo info', script_output('zypper lr -U'));
     if (is_transactional) {
-        trup_call('pkg install xfstests fio');
+        trup_call('pkg install xfstests');
```
[the remark this answers is not in the document]
Comment (SeroSun) [reply tests/xfstests/install.pm:48]: IMO, the package is somehow stable, not changing frequently. Let's modify it if things change.

**Review by mloviska — APPROVED** [review]

**Pull request merged**